app/src/main/res/values/donottranslate_questionnaire_values.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <string-array name="questionnaire_values_age">
+        <item>10代</item>
+        <item>20代</item>
+        <item>30代</item>
+        <item>40代</item>
+        <item>50代</item>
+        <item>60代</item>
+        <item>答えたくない</item>
+    </string-array>
+    <string-array name="questionnaire_values_android_user">
+        <item>なし</item>
+        <item>〜半年未満</item>
+        <item>半年〜1年未満</item>
+        <item>1年〜2年未満</item>
+        <item>2年〜3年未満</item>
+        <item>3年〜4年未満</item>
+        <item>4年〜5年未満</item>
+        <item>5年以上</item>
+    </string-array>
+    <string-array name="questionnaire_values_android_developer">
+        <item>なし</item>
+        <item>〜半年未満</item>
+        <item>半年〜1年未満</item>
+        <item>1年〜2年未満</item>
+        <item>2年〜3年未満</item>
+        <item>3年〜4年未満</item>
+        <item>4年〜5年未満</item>
+        <item>5年以上</item>
+    </string-array>
+    <string-array name="questionnaire_values_job_category">
+        <item>エンジニア（Android）</item>
+        <item>エンジニア（非Android）</item>
+        <item>企画・マーケティング</item>
+        <item>デザイナー</item>
+        <item>営業</item>
+        <item>人事</item>
+        <item>メディア関係</item>
+        <item>教育関係</item>
+        <item>学生</item>
+        <item>答えたくない</item>
+        <item>__other_option__</item>
+    </string-array>
+    <string-array name="questionnaire_values_job_title">
```

Comment:
**[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.array.questionnaire_values_job_title appears to be unused [UnusedResources]